QueueModel.py

```diff
@@ -48,6 +48,13 @@ def __getstate__(self):
     def __setstate__(self, d):
         self.__dict__.update(d)
 
+    def __getstate__(self):
+        d = dict(self.__dict__)
+        return d
+
+    def __setstate__(self, d):
```

Comment:
Hmm, not sure. Also in the final code is defined 3 times. Ok, I will accept the PR and then check the final version